lib/services/elasticsearch.js

```diff
@@ -687,6 +687,58 @@ class ElasticSearch extends Service {
     }
   }
 
+  /**
+  * Updates all documents matching the provided filters
+  *
+  * @param {String} index - Index name
+  * @param {String} collection - Collection name
+  * @param {Object} query - Query to match documents
+  * @param {Object} changes - Changes wanted on documents
+  * @param {Object} options - from (undefined), size (undefined), refresh (undefined), timeout (undefined), userId (undefined)
+  *
+  * @returns {Promise.<Object>} { documents, total, failures: [ _shardId, reason ] }
+  */
+  async updateByQuery(
+    index,
+    collection,
+    query,
+    changes,
+    { refresh, from, size } = {})
```

Comment:
`from` and `size` are undocumented and unused by this function callers

Reply:
Actually `size` is used by `_getAllDocumentsFromQuery`, and in the `deleteByQuery` method it has a default value of `1000`, so IMHO this method should have the same default value and no `from` argument as @scottinet says.   
See https://github.com/kuzzleio/kuzzle/blob/master/lib/services/elasticsearch.js#L629  
(The `from` option should be removed from the `deleteByQuery` method also)